README.md

```diff
@@ -46,23 +46,23 @@ client.quota.callback.static.storage.per.volume.limit.min.available.bytes=536870
-| client.quota.callback.static.storage.check-interval                       | Integer | 60                      | Interval between storage check runs (in seconds, a value of 0 means disabled)                                                                                                  |
-| client.quota.callback.static.storage.per.volume.limit.min.available.bytes | Long    | None                    | Stop message production if availableBytes <= this value. Mutually exclusive with `client.quota.callback.static.storage.per.volume.limit.min.available.ratio`.                 |
-| client.quota.callback.static.storage.per.volume.limit.min.available.ratio | Double  | None                    | Stop message production if availableBytes / capacityBytes <= this value. Mutually exclusive with `client.quota.callback.static.storage.per.volume.limit.min.available.bytes`. |
+| Property                                                                  | Type    | Default                 | Description                                                                                                                                                                                                 |
+|---------------------------------------------------------------------------|---------|-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| client.quota.callback.static.produce                                      | Double  | 1.7976931348623157e+308 | Produce bandwidth rate quota (in bytes)                                                                                                                                                                     |
+| client.quota.callback.static.fetch                                        | Double  | 1.7976931348623157e+308 | Consume bandwidth rate quota (in bytes)                                                                                                                                                                     |
+| client.quota.callback.static.request                                      | Double  | 1.7976931348623157e+308 | Request processing time quota (in seconds)                                                                                                                                                                  |
+| client.quota.callback.static.excluded.principal.name.list                 | String  | ""                      | Semicolon-separated list of principals excluded from the quota. Each principal have to be prefixed with `User:` prefix to exclude it from the quota.                                                        |
```

Comment:
```suggestion
| client.quota.callback.static.excluded.principal.name.list                 | String  | ""                      | Semicolon-separated list of principals excluded from the quota. Each principal has to be prefixed with `User:`.                                                        |
```
have -> has
"to exclude it from the quota" is repetitious and slightly confusing.

Reply:
Ah yes, thanks 😄